posthog/hogql/test/test_printer.py

```diff
@@ -1212,7 +1212,7 @@ def test_print_global_settings(self):
         )
         self.assertEqual(
             printed,
-            f"SELECT 1 FROM events WHERE equals(events.team_id, {self.team.pk}) LIMIT 10000 SETTINGS readonly=2, max_execution_time=10, allow_experimental_object_type=1, format_csv_allow_double_quotes=0, max_ast_elements=1000000, max_expanded_ast_elements=1000000",
+            f"SELECT 1 FROM events WHERE equals(events.team_id, {self.team.pk}) LIMIT 10000 SETTINGS readonly=2, max_execution_time=10, allow_experimental_object_type=1, format_csv_allow_double_quotes=0, max_ast_elements=1000000, max_expanded_ast_elements=1000000, max_query_size=524288",
```

Comment:
I am not sure why I needed to add this but it was failing locally for me on `master` too. Probably worth a second opinion from someone on @PostHog/team-product-analytics

Reply:
Apologies, we merged in some unsnapshotted changes. These should be reverted by now (merge with master)